Aztec/Classes/Extensions/String+RangeConversion.swift

```diff
@@ -5,13 +5,16 @@ import Foundation
 //
 extension String
 {    
-    func rangeFromNSRange(nsRange : NSRange) -> Range<String.Index>? {
-        let from16 = utf16.startIndex.advancedBy(nsRange.location, limit: utf16.endIndex)
-        let to16 = from16.advancedBy(nsRange.length, limit: utf16.endIndex)
-        if let from = String.Index(from16, within: self),
-            let to = String.Index(to16, within: self) {
+    func rangeFromNSRange(_ nsRange : NSRange) -> Range<String.Index>? {
+        
+        let rangeStartIndex = utf16.startIndex.advanced(by: nsRange.location)
+        let rangeEndIndex = rangeStartIndex.advanced(by: nsRange.length)
```

Comment:
Just for info - I removed it because I don't think it should be a problem.  Let's keep our eyes open for this.